subprojects/docs/src/docs/userguide/publishingMaven.adoc

```diff
@@ -175,13 +175,32 @@ If a project defines multiple publications then Gradle will publish each of thes
 Publications are published to repositories. The repositories to publish to are defined by the api:org.gradle.api.publish.PublishingExtension#getRepositories()[] container.
 
 ++++
-<sample dir="maven-publish/quickstart" id="publishing_maven:repositories" title="Declaring repositories to publish to">
+<sample dir="maven-publish/quickstart" id="publishing_maven:example:repositories" title="Declaring repositories to publish to">
     <sourcefile file="build.gradle" snippet="repositories"/>
 </sample>
 ++++
 
 The DSL used to declare repositories for publication is the same DSL that is used to declare repositories to consume dependencies from, api:org.gradle.api.artifacts.dsl.RepositoryHandler[]. However, in the context of Maven publication only api:org.gradle.api.artifacts.repositories.MavenArtifactRepository[] repositories can be used for publication.
 
+[[publishing_maven:snapshot_and_release_repositories]]
+==== Snapshot and release repositories
+
+It is a common practice to publish snapshots and releases to different Maven repositories. A simple way to accomplish this is to configure the repository URL based on the project version. The following sample uses one URL for versions that end with “`SNAPSHOT`” and a different URL for the rest:
```

Comment:
“`SNAPSHOT`” <- There's no reason to make this monospace, particularly with quotes. Also, please use normal quotes, not smart ones. Try: "SNAPSHOT"

Reply:
Will do! Out of curiosity: Why not smart quotes?

Reply:
Good question. I will justify my stance with: consistency :smile:

If I'm honest, I'm not a fan of curly quotes. But I also think it's awkward to use curly quotes in prose and straight quotes in code when you're using the same editor for both.